tests/src/test/java/alluxio/client/cli/fs/command/LoadMetadataCommandIntegrationTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fs.command;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystemTestUtils;
+import alluxio.client.file.URIStatus;
+import alluxio.client.cli.fs.AbstractFileSystemShellTest;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.DeletePOptions;
+import alluxio.grpc.GetStatusPOptions;
+import alluxio.grpc.LoadMetadataPType;
+import alluxio.grpc.WritePType;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * Tests for loadMetadata command.
+ */
+public final class LoadMetadataCommandIntegrationTest extends AbstractFileSystemShellTest {
+  @Test
+  public void loadMetadataDir() throws IOException, AlluxioException {
+    String dirPath = "/testRoot/layer1/layer2/layer3/";
+    String filePathA = dirPath + "testFileA";
+    String filePathB = dirPath + "testFileB";
+    FileSystemTestUtils
+        .createByteFile(sFileSystem, filePathA, WritePType.CACHE_THROUGH, 10);
+    FileSystemTestUtils
+        .createByteFile(sFileSystem, filePathB, WritePType.THROUGH, 30);
```

Comment:
Why creating two files with different WriteType? If you want to validation all write types, maybe add files for each of them?